docs/source/admin/upgrading.rst

```diff
@@ -63,11 +63,11 @@ Upgrade JupyterHub packages
 
 There are two environments where the ``jupyterhub`` package is installed:
 
-#. The *hub environment*, which is where the JupyterHub server process
+#. The *hub environment*: Which is where the JupyterHub server process
```

Comment:
A colon would be _instead of_ `which is...`:

```suggestion
#. The *hub environment*: where the JupyterHub server process
```

We also don't typically capitalize after a colon unless it's a full sentence (which this isn't).

Reply:
I have made changes on this @minrk